views/partials/sign_up_banner.hbs

```diff
@@ -0,0 +1,14 @@
+<div id="sign-up-banner">
+  <div class="section-wrapper">
+    <div class="fourth subscribe-icon-wrapper">
+      <img src="img/subscribe-icon.svg" />
+    </div>
+    <div class="half">
+      <span class="medium">Subscribe to Firefox Monitor</span>
+      <p class="demi">Subscribe to Firefox Monitor for a full report on your compromised accounts (including sensitive sites and spam lists).</p>
```

Comment:
Do we include sensitive sites and spam lists? Or not on the site and only if you subscribe?

Reply:
We don't show the sensitive breaches. We can include them in the "Welcome to Firefox Monitor" email.

Reply:
OK, so we'd include past sensitive breaches in the welcome email, and then email them in near-real-time on future sensitive breaches (but still never show them on the site)?

Reply:
Correct.